### ISSUE TITLE
`pause` and `resume` are never called in Android live wallpapers

### ISSUE THREAD

**Issue by chengyuhui** [libgdx/libgdx#6874]
Please ensure you have given all the following requested information in your report.

#### Issue details
The reference here is extremely old and device-specific. Since this looks like a bug in the system, it is likely solved after all these years (I can not trigger such errors with all my Android devices).
https://github.com/libgdx/libgdx/blob/79cf00af53b7f38667291fbacf544d3074a811bd/backends/gdx-backend-android/src/com/badlogic/gdx/backends/android/AndroidLiveWallpaper.java#L95-L111

My particular use case is that the application can send commands to the wallpaper, which is in the background and does not have a GL context. Since some of the commands require loading textures, they need to be enqueued until there is a context.

However,  the state of the application is only ever communicated to the `ApplicationAdapter` when the graphics is paused, which never happens.
https://github.com/libgdx/libgdx/blob/79cf00af53b7f38667291fbacf544d3074a811bd/backends/gdx-backend-android/src/com/badlogic/gdx/backends/android/AndroidGraphics.java#L474-L484

I might be able to determine that the GL surface is not available in other ways, but can not receive an event when it becomes available. This is also very confusing since it is not documented.

If pausing the graphics is still undesired, maybe we can at least call these lifecycle methods somewhere?
#### Reproduction steps/code
N/A

#### Version of libGDX and/or relevant dependencies
1.10.0

#### Stacktrace
N/A

#### Please select the affected platforms
- [x] Android
- [ ] iOS
- [ ] HTML/GWT
- [ ] Windows
- [ ] Linux
- [ ] macOS

**Comment by AndroidDeveloperLB** [comment]
I actually do get a call to "resume", but not for "pause".
Workaround that might be useful (though the callbacks are called sometimes more than they should):

```
    override fun onCreateEngine(): Engine {
        return object : AndroidWallpaperEngine() {
            override fun onVisibilityChanged(visible: Boolean) {
                super.onVisibilityChanged(visible)
                Log.d("AppLog", "onVisibilityChanged:$visible")
            }

            override fun onPause() {
                super.onPause()
                Log.d("AppLog", "onPause")
            }

            override fun onResume() {
                super.onResume()
                Log.d("AppLog", "onResume")
            }
        }
    }
```

Also wrote about it here:
https://github.com/kloverde/android-LibGdxLiveWallpaperTemplate/issues/2

**Comment by chengyuhui** [comment]
Yes, `resume` is actually being called. `onPause` getting called multiple times is actually fine for me, as long as it is not called if the application has already been resumed. I only need a flag to delay the loading of textures.
